### DISCUSSION TITLE
inChangeNotice dynamic values

### DISCUSSION THREAD

**Discussion by karel-debrabandere-bosa-fgov-be** [OP-TED/eForms-SDK#227] [Q&A] [answered]
The dynamic values for this field attribute are not following the documented dynamic property structure:

The dynamic value for the attribute inChangeNotice in field `"BT-262-Procedure"` is `{ "canModify" : false }`, while it should be `{ "canAdd" : true, "canModify" : false, "canRemove" : true }`.

**Comment by YvesJo** [comment, marked as answer]
BT-262-Procedure is just mandatory and not repeatable, therefore it is neither possible to add or remove BT-262-Procedure with a Change, and canAdd or canRemove do not add any value.
